charts/kubescape-operator/templates/node-agent/daemonset.yaml

```diff
@@ -19,6 +19,8 @@ spec:
       tier: {{ .Values.global.namespaceTier }}
   template:
     metadata:
+      annotations:
+        container.apparmor.security.beta.kubernetes.io/node-agent: unconfined
```

Comment:
@slashben can you please explain what is this annotation?

Reply:
It removes the restrictions of AppArmor (on nodes that has AppArmor) from the `node-agent` container. This is needed since AppArmor blocks the mounting of `/sys/kernel/tracing` and limits access to `/proc/<pid>`. Beforehand, `privileged: true` bypassed AppArmor therefore we didn't need this. Now we do.